### PR TITLE
orthogonal circle, analysis file

### PR DESCRIPTION
**Pull request by AmirSahrani**

### CODE REVIEW

File: Engels_14947994_Sahrani_12661651_Siregar_1486305.py [hunk 1]
```diff
@@ -1,6 +1,7 @@
 import numpy as np
 import pandas as pd
 from scipy import stats
+from scipy.stats import qmc
```
Comment (nabilasiregar) [inline Engels_14947994_Sahrani_12661651_Siregar_1486305.py:4]: remove this

File: Engels_14947994_Sahrani_12661651_Siregar_1486305.py [hunk 2]
```diff
@@ -272,6 +291,11 @@ def confidence_intervals(filename, alpha):
         
         posthoc_result = pg.pairwise_gameshowell(data=data, dv="area", between="method")
         print(posthoc_result)
+        sampler = qmc.Sobol(d=2)
```
Comment (nabilasiregar) [inline Engels_14947994_Sahrani_12661651_Siregar_1486305.py:294]: and lines related to sobol

File: Engels_14947994_Sahrani_12661651_Siregar_1486305.py (hunk 1 above)
Comment (AmirSahrani) [reply Engels_14947994_Sahrani_12661651_Siregar_1486305.py:4]: This one is also needed for the ortho circle sample, but I've removed the other lines

**Review by AmirSahrani — COMMENTED** [review]
done? (never used this review thing)

**Review by AmirSahrani — COMMENTED** [review]
quick update to make names in analysis.py more clear

**Pull request merged**